### DISCUSSION TITLE
Welcome to OnePlace Discussions!

### DISCUSSION THREAD

**Discussion by atharvashirdhankar** [StarTrooper08/OnePlace#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello everyone, mysef Sumon

I have been learning code on and off since 2019, learnt few things now I'm trying to get into open source , so here I'm to connect , learn and make open source contributions

**Reply by atharvashirdhankar** [reply]
Hi @sumonchhetry you can hope onto the improve documentation issue for your first contribution .
